### PR TITLE
Fix minor formatting issue in kubectl docs

### PR DESCRIPTION
**Pull request by syaffers**
When visiting the kubectl docs, the table present at https://kubernetes.io/docs/reference/kubectl/#operations contains a row that describes the "top" command but displays broken or incorrect information.

I found that this problem persisted in other localizations as well.

This PR fixes the broken formatting for the [en] and [zh-cn] docs, and updates the [jp] and [ko] docs.

### CODE REVIEW

**Comment by T-Lakshmi** [comment]
Hi @syaffers,
Thanks for Raising PR! This PR having changes in multiple languages. We usually don't merge PRs impacting multiple languages at once. I would recommend modifying only the primary website content (English content)in this PR. And after this English PR merge, submit separate PRs for other languages.

**Comment by syaffers** [comment]
@T-Lakshmi Noted. Reverted [ko], [ja], and [zh-cn].

**Review by Gauravpadam — COMMENTED** [review]
[Preview](https://deploy-preview-43853--kubernetes-io-main-staging.netlify.app/docs/reference/kubectl/#operations) looks good

lgtm, Thanks for the PR!

**Comment by T-Lakshmi** [comment]
[Preview page](https://deploy-preview-43853--kubernetes-io-main-staging.netlify.app/docs/reference/kubectl/#operations)
Looking Good to me,
Thanks.

**Comment by Gauravpadam** [comment]
(For ref)

Before:
image



After:
image

**Comment by windsonsea** [comment]
The [preview](https://deploy-preview-43853--kubernetes-io-main-staging.netlify.app/docs/reference/kubectl/) looks good.
/lgtm

**Review by kundan2707 — COMMENTED** [review]
/lgtm
Thanks

**Review by ydFu — APPROVED** [review]
/lgtm

**Review by divya-mohan0209 — COMMENTED** [review]
Thanks for the fix, @syaffers !
/approve

**Pull request merged**
